bentoml/cli/bento.py

```diff
@@ -125,22 +115,29 @@ def get(bento, limit, filters, output):
 
             _print_bentos_info(list_bento_versions_result.bentos, output)
 
-    @bento_repo.command(name='list', help='List BentoServices information')
+    @cli.command(name='list', help='List BentoServices information')
     @click.option(
-        '--limit', type=click.INT, help='Limit how many resources will be retrieved'
+        '--limit', type=click.INT, help='Limit how many BentoServices will be retrieved'
     )
     @click.option(
-        '--filters',
-        type=click.STRING,
-        help='List resources containing the filter string in name',
+        '--offset', type=click.INT, help='How many BentoServices will be skipped'
     )
+    @click.option(
+        '--order-by', type=click.Choice(['created_at', 'name']), default='created_at',
+    )
+    @click.option('--ascending-order', is_flag=True)
```

Comment:
use "--asc/--desc" instead?  which are the same keywords found in SQL, look more familiar for people